.github/workflows/cd.yml

```diff
@@ -0,0 +1,74 @@
+name: CD
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Kibana
+      uses: actions/checkout@v2
+      with:
+        path: kibana
+        repository: elastic/kibana
+        fetch-depth: 0
+    - name: Use release
+      run: |
+        cd ./kibana
+        git checkout tags/v7.9.1 -b v7.9.1
```

Comment:
can we make version `v7.9.1` configurable? Does it need to be hardcoded?

Reply:
yeah, we should be able to get this out of `kibana.json` file, I will fix it in future PR.